packages/xo-server/src/xo-mixins/remotes.js

```diff
@@ -83,7 +83,19 @@ export default class {
 
   async testRemote(remote) {
     const handler = await this.getRemoteHandler(remote)
-    return handler.test()
+    const answer = await handler.test()
+    const { success, ...benchmark } = answer
```

Comment:
Do you think we'll think about updating this if one day we decide to return more values from `test()`?

Reply:
Don't you have a good memory?
you're right, i change it.